step4/server.js

```diff
@@ -0,0 +1,78 @@
+const express = require('express');
+const fs = require('fs');
+const app = express();
+
+PORT = 5000;
+
+const getItems = () => {
+    const getItems = JSON.parse(fs.readFileSync('./data.json','utf-8'));
+    return getItems;
+}
+
+const save = (items) => {
+    fs.writeFileSync('./data.json',JSON.stringify(items),'utf-8');
+}
+
+function updateIdx(items) {
+    for(let i =0;i<items.length;++i) {
+        items[i].idx = i + 1; 
+    }
+}
+
+app.use(express.json());
+app.use(express.static(__dirname+'/dist'));
```

Comment:
static 등록 잘 해주셨네요!
```suggestion
app.use(express.static(`${__dirname}/dist`));
```
이렇게 템플릿 리터럴을 이용해서 표현할 수도 있답니다~